app/views/cars/_slot.html.erb

```diff
@@ -1,4 +1,9 @@
-
+<%
+=begin%>
+  You have all views in one place, like for car and even for slots, is there a reason for this?
+  below comparision is fine for now, but make sure you do not use it for comparing numbers
```

Comment:
All views are in one place because they are all partials. I felt that if they were individual views then, they deserved a folder of their own.